documentation.html

```diff
@@ -43,7 +43,111 @@ <h1>Components</h1>
+                    <div class="avatar">
+                        <img class="ava-sq-img avatar-xl" src="./assets/avatar.png" alt="Avatar" /></div>
+                    <div class="avatar">
+                        <img class="ava-sq-img avatar-l" src="./assets/avatar.png" alt="Avatar" /></div>
+                    <div class="avatar">
+                        <img class="ava-sq-img avatar-m" src="./assets/avatar.png" alt="Avatar" /></div>
+                    <div class="avatar">
+                        <img class="ava-sq-img avatar-s" src="./assets/avatar.png" alt="Avatar" /></div>
+                </div>
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28133%2C122%2C53%2C1%29&t=night-owl&wt=sharp&l=auto&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522ava-sq-img%2520avatar-xxl%2522%2520src%253D%2522%2522%2520alt%253D%2522Avatar%2522%2520%252F%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522ava-sq-img%2520avatar-xl%2522%2520src%253D%2522%2522%2520alt%253D%2522Avatar%2522%2520%252F%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522ava-sq-img%2520avatar-l%2522%2520src%253D%2522%2522%2520alt%253D%2522Avatar%2522%2520%252F%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522ava-sq-img%2520avatar-m%2522%2520src%253D%2522%2522%2520alt%253D%2522Avatar%2522%2520%252F%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522ava-sq-img%2520avatar-s%2522%2520src%253D%2522%2522%2520alt%253D%2522Avatar%2522%2520%252F%253E%250A%253C%252Fdiv%253E"
+                    style="width: 564px; height: 393px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+            <!-- Avatar Component Ends -->
 
+            <!-- ALert Component Starts -->
+            <div class="alert-component">
+                <h3>Alerts</h3>
+                <p>Alerts are used to attract user's attention for important information without interrupting the user's
+                    flow.</p>
+                <p>Alerts are available in 4 types- Primary alert, Success alert, Error alert, Warning alert and in two
+                    different styles: Solid Alert and Outline Alert.
+                    <h5>Examples of Solid Alert Box:</h5>
+                    <div class=" container alert-container">
+                        <div class="alert-item-container alert-primary-a">
```

Comment:
you would have used short class names like instead of alert-item-container you can use alert-item or just alert